TheForceEngine/TFE_DarkForces/player.cpp

```diff
@@ -1830,69 +1830,76 @@ namespace TFE_DarkForces
 			//s_282344 = 34 - dH;
 		}
 
```

Comment:
This diff below looks big but all of the lines from this point to `:1844` are unchanged except for the following:
* I removed the `if` block that starts on line `:1836` and thus all of the code within that block down to `:1885` was indented left-ward (hence why the diff looks large). I removed it because we now need to run these calculations regardless of whether head waving is on since these are used for weapon waving.
* Any assignments to `s_headwaveVerticalOffset` in this range of code now assign to a new local variable `headwaveVerticalOffset` (see `:1835`). I essentially avoid modifying `s_headwaveVerticalOffset` until I know the head waving is on and needs to be applied since modifying `s_headwaveVerticalOffset` alters how player projectiles (like lasers from the blaster) are calculated.